layouts/_default/baseof.html

```diff
@@ -19,6 +19,14 @@
     <header class="app-header">
       <a href="{{ .Site.BaseURL }}"><img class="app-header-avatar" src="{{ .Site.Params.avatar | default "avatar.jpg" | relURL }}" alt="{{ .Site.Params.author | default "John Doe" }}" /></a>
       <h1>{{ .Site.Title }}</h1>
+      {{- with .Site.Menus.main }}
+      <nav class="app-header-menu">
+        -
+        {{- range . }}
+          <a href="{{ .URL }}">{{ .Name }}</a> -
+        {{- end }}
```

Comment:
```suggestion
        {{- range . }}
          <a href="{{ .URL }}">{{ .Name }}</a><br>
        {{- end }}
```

For my personal taste it looks better like that